docs/aws-deployment.md

```diff
@@ -1,11 +1,10 @@
 # Data Loch AWS deployment recipe
 
-Data Loch is an application that runs scheduled scripts to sync latest canvas data dumps (recovered from Canvas Data API) and refreshes cnavas table housed institutional data lake. The application uses AWS services like S3, Elastic Beanstalk and Redshift that are assembled together to create a highly available
-platform for running institutional analytics.
+Data Loch is an application that runs scheduled scripts to sync latest canvas data dumps (recovered from Canvas Data API) and refreshes Canvas tables housed in the institutional data lake. The application uses AWS services like S3, Elastic Beanstalk and Redshift Spectrum that are assembled together to create a highly available platform for running institutional analytics.
```

Comment:
@sandeepmjay The text above reads like an executive summary. I recommend we move it to the top of README. One suggestion: describe Loch's purpose before describing implementation.